src/olympia/addons/serializers.py

```diff
@@ -13,8 +14,8 @@
 
 class FileSerializer(serializers.ModelSerializer):
     url = serializers.SerializerMethodField()
-    platform = serializers.ReadOnlyField(source='get_platform_display')
-    status = serializers.ReadOnlyField(source='get_status_display')
+    platform = ReverseChoiceField(choices=amo.PLATFORM_CHOICES_API.items())
+    status = ReverseChoiceField(choices=amo.STATUS_CHOICES_API.items())
```

Comment:
Fields are instantiated immediately so it would only save a few microseconds each time a new worker is started, so I didn't see much of a benefit to create the store the reversed dicts in constants.